packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -23,6 +23,14 @@ export const testParasRococo: EndpointOption[] = [
       Arctic: 'wss://arctic-rococo-rpc.icenetwork.io'
     }
   },
+  {
+    info: 'rococoAmplitude',
+    paraId: 2124,
+    text: 'Amplitude',
+    providers: {
+      PendulumChain: 'wss://pencol-roc-00.pendulumchain.tech'
+    }
+  },
```

Comment:
This is indeed missing (also clueless as to why - all I can think of is a conflict and unintentional drop along the way) -

https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/endpoints/testingRelayRococo.ts

Reply:
Hey Jaco, I think our forked master diverged from the upstream, and from that point it is broken. I created another PR that should be the valid one. Will close this with a comment below linking it.